content/docs/0.8.1/snapshots-and-backups/backup-and-restore/backupstores-and-backuptargets.md

```diff
@@ -5,19 +5,23 @@
 
 A backupstore is a NFS server or S3 compatible server.
 
-A backup target represents a backupstore in Longhorn. The backup target can be set at `Settings/General/BackupTarget`
+A backup target is the endpoint of a backupstore in Longhorn. The backup target can be set at `Settings/General/BackupTarget`
 
 See [here](#set-backuptarget) for details on how to setup backup target.
 
 Longhorn also supports setting up recurring snapshot/backup jobs for volumes, via Longhorn UI or Kubernetes Storage Class. See [here](../../scheduling-backups-and-snapshots) for details.
 
 ## Set BackupTarget
 
-The user can setup a S3 or NFS type backupstore to store the backups of Longhorn volumes.
+In this section, you'll learn how to store the backups of Longhorn volumes. Multiple backup options are available:
 
-If the user doesn't have access to AWS S3 or want to give a try first, we've also provided a way to [setup a local S3 testing backupstore](#setup-a-local-testing-backupstore) using [Minio](https://minio.io/).
+- [AWS S3 backupstore](#set-up-aws-s3-backupstore)
+- [Local testing backupstore with Minio S3 server](#set-up-a-local-testing-backupstore)
+- [NFS backupstore](#nfs-backupstore)
 
-### Setup AWS S3 backupstore
+If the user doesn't have access to AWS S3 or want to give a try first, we've also provided a way to [setup a local S3 testing backupstore](#set-up-a-local-testing-backupstore) using [Minio](https://minio.io/).
```

Comment:
Repeated the second items above?

Reply:
combined with the section above, removed redundancy